CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ## Version 0.3.0 (In Development)
 
+- Redesign factories, batch runs in BatchedFactory, fix Qiskit utils tests (@rmlarose, @andreamari, gh-381).
```

Comment:
this is in the wrong spot -- will need to resolve merge conflicts when moving to "All Changes" section